src/incrementer.js

```diff
@@ -228,28 +239,36 @@ var Incrementer = {
      * @param {jQueryElement} $elt - plugin's element
      * @fires Incrementer#minus.incrementer
      */
-    _dec: function($elt) {
+    _dec: function _dec($elt) {
         var options = $elt.data(dataNs),
             current = parseFloat($elt.val() || 0),
             value;
-
         value = gamp.sub(current, options.step);
 
+        if ($elt[0].getAttribute('name') === 'normalMaximum') {
+            options.max = parseFloat($elt.parent().parent().find("[name='normalMaximum']").attr('value')) - 1;
+            options.min = parseFloat($elt.parent().parent().find("[name='normalMinimum']").attr('value'));
+        } else {
+            options.max = parseFloat($elt.parent().parent().find("[name='normalMaximum']").attr('value'));
+            options.min = parseFloat($elt.parent().parent().find("[name='normalMinimum']").attr('value')) - 1;
+        }
+
+        if (options.max < options.min) {
+            return;
+        }
+
         if (options.zero === true && _.isNumber(options.min) && value < options.min) {
             value = 0;
         }
 
-        if (
-            options.min === null ||
-            (_.isNumber(options.min) && value >= options.min) ||
-            (options.zero === true && value === 0)
-        ) {
+        if (options.min === null || _.isNumber(options.min) && value >= options.min || options.zero === true && value === 0) {
             $elt.val(value);
-
+            $elt[0].setAttribute('value', value);
```

Comment:
Why do we have 2 ways to set value?

Reply:
Because I use one for the current behavior to fix the bug and the other remains to avoid regressions